src/instruments/buildSrc/igniter/tasks.mjs

```diff
@@ -34,9 +34,19 @@ const ecamPages = [
     },
 ];
 
-function getInputs() {
-    const baseInstruments = fs.readdirSync(join(Directories.instruments, 'src'), { withFileTypes: true })
-        .filter((d) => d.isDirectory() && fs.existsSync(join(Directories.instruments, 'src', d.name, 'config.json')));
+export function getInputs() {
+    const baseInstruments = fs.readdirSync(`${Directories.instruments}/src`, { withFileTypes: true })
+        .filter((d) => d.isDirectory() && fs.existsSync(`${Directories.instruments}/src/${d.name}/config.json`));
+
+    return [
+        ...baseInstruments.map(({ name }) => ({ path: name, name, isInstrument: true })),
+        ...ecamPages.map((def) => ({ ...def, isInstrument: false })),
+    ];
+}
+
+export function getInstrumentsIgniterTasks() {
+    const baseInstruments = fs.readdirSync(`${Directories.instruments}/src`, { withFileTypes: true })
+        .filter((d) => d.isDirectory() && fs.existsSync(`${Directories.instruments}/src/${d.name}/config.json`));
```

Comment:
**issue**: This breaks building stuff on Windows.
**suggestion**: Please use `path.join` as per #5278 (and the code before you made the change).

Reply:
If we really want to support it we need CI for it.

Reply:
Without it using watch mode on Windows is just not going to work.
Using `path.join` is also just good practice which should always be applied to nodejs code 😀.

Reply:
Plus, I made all the changes to make it work. So it's just nice if it's not broken again.

Reply:
Yeah, I understand that. Just saying that we should look at CI so it doesn't get broken accidentally again. :)